tpp-prophets/interprophet.xml

```diff
@@ -1,59 +1,31 @@
 <tool id="proteomics_search_interprophet_1" name="InterProphet" version="1.1.1">
     <description>Combine Peptide Prophet results from multiple search engines</description>
     <requirements>
-        <container type="docker">iracooke/protk-1.4.3</container>
         <requirement type="package" version="1.4.4a">protk</requirement>
     </requirements>
-    <command>
+    <command detect_errors="aggressive">
 <![CDATA[
         export XML_ONLY="true";
 
         interprophet.rb --galaxy
-        #if str( $input_type.type ) == "files":
-            #for $pepxml_file in $input_type.pepxml_files:
-                '${pepxml_file}'
-            #end for
-        #else
-            #for $list_item in $input_type.pepxml_list:
-                '${list_item}'
-            #end for
-            #for $extra_list_holder in $input_type.extra_pepxml_lists:
-                #for $list_item in $extra_list_holder.extra_list:
-                    '${list_item}'
-                #end for
-            #end for
-        #end if
-
-            -o interprophet_output.pep.xml
-            $use_nss
-            $use_nrs
-            $use_nse
-            $use_nsi
-            $use_nsm
-            --p-thresh $p_thresh
-            --threads "\${GALAXY_SLOTS:-12}"
+        #for $input_file in $input_files:
```

Comment:
Are you sure this does the same thing?  The old code was necessary to allow for Galaxy's multifile input support

Reply:
As I understand the code before the only use-case for the old one was that you wanted to take multiple collections and "merge" them, correct? This should be handled by one of the build-in collection tools.